android/modules/network/src/java/ti/modules/titanium/network/TiHTTPClient.java

```diff
@@ -350,15 +350,10 @@ private void handleEntityData(byte[] data, int size, long totalSize, long conten
 		KrollDict callbackData = new KrollDict();
 		callbackData.put("totalCount", contentLength);
 		callbackData.put("totalSize", totalSize);
-		callbackData.put("size", size);
+		callbackData.put(TiC.PROPERTY_SIZE, size);
 
-		byte[] blobData = new byte[size];
-		System.arraycopy(data, 0, blobData, 0, size);
-
-		TiBlob blob = TiBlob.blobFromData(blobData, contentType);
-		callbackData.put("blob", blob);
```

Comment:
Technically this is a breaking Api change (and should go into 8.0.0) since this is removed. Although, it's hard to see how many people would be using this undocumented event callback property...

Reply:
It is. But from searching our repo's and JIRA's history, it doesn't appear to be used. We've discussed it internally in the office and decided to remove it.

Also note that we have a customer that wants a solution to this performance issue before Titanium 8.0.0. You may want to talk to Eric about this.